src/components/List/AddCard.js

```diff
@@ -0,0 +1,198 @@
+import Snackbar from "../Snackbar";
+
+const useStyles = makeStyles((theme) => ({
+  root: {
+    backgroundColor: "white",
+    maxWidth: 500,
+    width: "100%",
+    height: "100%",
+  },
+  buttons: {
+    display: "flex",
+    justifyContent: "space-between",
+  },
+  dialogTitle: {
+    textAlign: "center",
+    "& h2": {
+      fontSize: "2rem",
+      fontWeight: 500,
+    },
+  },
+  textField: {
+    display: "flex",
+    paddingBottom: theme.spacing(2),
+  },
+  formControl: {
+    display: "flex",
+    paddingBottom: theme.spacing(2),
+  },
+}));
+
+export default function AddCard({ open, close, cardTitle, list, triggerUpdate }) {
+  const classes = useStyles();
+  const [selectedGroup, setSelectedGroup] = useState("");
+  const [selectedUser, setSelectedUser] = useState("");
+  const [selectedDate, setSelectedDate] = useState("");
+  const [title, setTitel] = useState("");
+  const [userData, isUserData] = useCurrentUser();
+  const [isLoading, setIsLoading] = useState(true);
+  const [allUserInGroup, setAllUserInGroup] = useState();
+  const [snackbarContent, setSnackbarContent] = useState();
+
+  useEffect(() => {
+    const getUserInGroup = async () => {
+      setIsLoading(true);
+      const allUserData = await getAllUserData();
+      const groupUserNames = allUserData
+        .filter((user) => Object.keys(user.groups).some((id) => id === selectedGroup))
+        .map((user) => ({
+          name: user.userName,
+          id: user.userId,
+        }));
+      setAllUserInGroup(groupUserNames);
+      setIsLoading(false);
+    };
+    getUserInGroup();
+  }, [selectedGroup]);
+
+  const handleDropDown = (e) => {
+    setSelectedGroup(e.target.value);
+  };
+
+  const handleSelectUser = (e) => {
+    setSelectedUser(e.target.value);
+  };
+
+  const handleSelectDate = (e) => {
+    setSelectedDate(e.target.value);
+  };
+
+  const handleTitel = (e) => {
+    setTitel(e.target.value);
+  };
+
+  const handleSave = async (e) => {
+    if (title && selectedDate && selectedGroup && selectedUser) {
+      await addListEntry({
+        title,
+        date: selectedDate,
+        groupID: selectedGroup,
+        assignedTo: selectedUser,
+        list,
+      }); // Liste abstrahieren
+      setSnackbarContent({
+        message: "Dein Eintrag wurde erfolgreich erstellt.",
+        status: "success",
+        open: true,
+      });
+      triggerUpdate();
+      close();
+      setSelectedGroup("");
+      setTitel("");
+      setSelectedDate("");
+      setSelectedUser("");
+    } else {
+      setSnackbarContent({
+        message: "Fülle bitte alle Felder aus!",
+        status: "error",
+        open: true,
+      });
+    }
+  };
+
+  return (
+    <>
+      <Snackbar snackbarContent={snackbarContent} setSnackbarContent={setSnackbarContent} />
+      <Dialog open={open} onClose={close} className={classes.dialog}>
+        <DialogTitle id="filme-serien-hinzufügen" className={classes.dialogTitle} variant="h1">
```

Comment:
Komische id...
Ist vielleicht etwas groß oder? Vielleicht mal andere Variante ausprobieren :)

Reply:
Also in Zeile 124 :D